CMakeLists.txt

```diff
@@ -57,6 +57,11 @@ macro(wf_add_library _LIB_NAME _SOURCE_FILES_VAR _HEADER_FILES_VAR)
 
 endmacro()
 
+# Required for crc32 calculation
+find_package(ZLIB REQUIRED)
+include_directories(${ZLIB_INCLUDE_DIRS})
```

Comment:
Lol. I definatelly had that there, otherwise linker error would not have been gone. I have done some last minute cleanup which probably deleted it.